### ISSUE TITLE
✨ [Feature request] - Make the world map the home page and add walkthrough

### ISSUE THREAD

**Issue by edwardchalstrey1** [Seshat-Global-History-Databank/seshat#209]
### Description of Improvement

This has been discussed a while back, but as a concrete suggestion of what to do:

1. Navigating to the website directs you to the World Map page by default
2. There is a popup that appears over the centre of the map with a close button in the corner.
    - This popup should only appear the first time someone visits the website, and should say something like "Welcome to the Seshat Global History databank" and some of the other basic info from the current homepage like "the most current and comprehensive body of knowledge about human history..."
    - Could also include a walkthrough of the functionality, both for the World Map controls and the navbar items for browsing the rest of the site - see https://introjs.com/
3. For the existing homepage, I suggest we can either keep accessible by clicking the Seshat logo as currently, or remove it completely if it's not needed

### Dependencies

- #196 

### Definition of Done

- [ ] The feature has been developed on a feature branch.
- [ ] The feature branch has been tested in the Staging website.
- [ ] A pull request has been created for the feature branch to be merged into the dev branch.

**Comment by edwardchalstrey1** [comment]
@MajidBenam No rush, but I'm curious to know your thoughts/suggestions on this idea

**Comment by edwardchalstrey1** [comment]
On #221 I have added a simpler version of this by just updating the help popup and opening it by default

**Comment by MajidBenam** [comment]
Hi @edwardchalstrey1 I believe the world map could eventually serve as our landing homepage, but not at this stage. We still need to understand how users will respond to the way we’re currently showcasing the world map. Also, making the map the landing page might overshadow the other valuable aspects of Seshat.

From my experience with Seshat, we’re not quite ready to feature the world map as the central entry point. Putting maps as the landing page provokes people more, draws more attention and exposes our database to a wider audience. Given the current quality of our data (such as the number of uncoded or unknown records and the limited precision of our borders) it might not be ideal to present a data visualization, like the maps, as the very first thing the audience sees.

That said, there’s definitely room for improvement on the current landing page. However, we should manage our expectations. The landing pages for a project like Seshat may never be as attractive as those for other types of products.

What do you think?

**Comment by edwardchalstrey1** [comment]
@MajidBenam Makes sense, I agree! I will keep this issue open but this can be one for a future year
